multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Messages.java

```diff
@@ -14,6 +14,7 @@ public class Messages {
     public static final String ERROR_RETRIEVING_MTA_RESOURCE_CONTENT = "Error retrieving content of MTA resource \"{0}\"";
     public static final String ERROR_SERVICE_NEEDS_TO_BE_RECREATED_BUT_FLAG_NOT_SET = "Service described by MTA resource \"{0}\" of type [{1}] does not match already existing service \"{2}\" of type [{3}] and needs to be recreated. Use command line option \"--delete-services\" to enable the deletion of the existing one.";
     public static final String SIZE_OF_APP_EXCEEDS_MAX_SIZE_LIMIT = "The size of the application exceeds max size limit \"{0}\"";
+    public static final String ARCHIVE_SIGNATURE_VERIFICATION_IS_NOT_ENABLED = "Archive signature verification functionality is removed. Stop using command line option \"--verify-archive-signature\" or any other approach that enable it.";
```

Comment:
This message is quite intrusive and won't be liked by clients.
Maybe change it to `Archive signature verification functionality is discontinued. Please do not use the option "--verify-archive-signature".`